nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -85,15 +85,12 @@ def target_point(target_type: TargetType, target_node_name: str, port_id: int) -
 
     @staticmethod
     @abstractmethod
-    def bias_correction_command(target_point: TargetPoint,
-                                bias_value: np.ndarray,
-                                threshold: float) -> TransformationCommand:
+    def bias_correction_command(target_point: TargetPoint, bias_value: np.ndarray) -> TransformationCommand:
```

Comment:
Could you also change it for the BiasCorrection, please?

Reply:
Good catch! I will fix it.